src/vm.sh

```diff
@@ -55,14 +55,14 @@ function vm_mount()
 
 function vm_umount()
 {
-  local flag="$1"
-  local qemu_img_path="$2"
-  local mount_point_path="$3"
+  local flag="$2"
+  local qemu_img_path="$3"
+  local mount_point_path="$4"
```

Comment:
If you are not using the first parameter it's a sign you are passing extra stuff to the functions, avoid this. You can use `shift` to remove the first parameter before passing the rest here.